src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -43,10 +47,24 @@ public static PreSubmitCallbackResponse<SscsCaseData> processPostponementRequest
             return response;
         }
 
+        DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        List<Hearing> allHearings = sscsCaseData.getHearings();
+        Hearing latestHearing =  allHearings.isEmpty() ? null : allHearings.get(allHearings.size() - 1);
+        assert latestHearing != null;
```

Comment:
Probably best to throw an exception/log message containing more details (case id) and why this failed. It really helps when it comes to telemetry.

Reply:
Agreed. Although in fairness we should use AOP to handle proper logging all across the application.

Also, we have the date pattern again.

So both things are best handled as tech debt.

One more point for "tech debt day". Meanwhile, I'll mark this as resolved.